tests/drivers/flash_simulator/src/main.c

```diff
@@ -266,6 +269,15 @@ static void test_double_write(void)
 	zassert_equal(-EIO, rc, "Unexpected error code (%d)", rc);
 }
 
+static void test_get_erase_value(void)
+{
+	const struct flash_parameters *fp = flash_get_parameters(flash_dev);
+
+	zassert_equal(fp->erase_value, FLASH_SIMULATOR_ERASE_VALUE,
+		      "Eipected erase value %hhx",
```

Comment:
s/Eipected/Expected/.

Also `%x` not `%hhx`.  The value gets promoted to unsigned int in the call, no need for unusual length modifiers.